lib/taskbook.js

```diff
@@ -192,6 +250,62 @@ class Taskbook {
     return data;
   }
 
+  _filterOverdue(data) {
```

Comment:
I think we could do this date computation natively then why needs an external module.?